### ISSUE TITLE
Error: EISDIR: illegal operation on a directory

### ISSUE THREAD

**Issue by lmartins** [jspm/jspm-cli#1160]
I've updated JSPM recently and since I cant build my project, returning the following error:

```
▶ jspm bundle-sfx app build/js/app.js
     Building the single-file sfx bundle for app...

err  Error on fetch for npm:lodash@3.10.1 at file:///Users/luismartins/Sites/webipack.dev/src/wp-content/themes/wip-sparkling/jspm_packages/npm/lodash@3.10.1
	Error: EISDIR: illegal operation on a directory, read
         at Error (native)
```

Since I started getting this error i've updated the lodash package to see if it would make any difference, unfortunately it made no difference.

```
▶ jspm -v
0.16.10
Running against local jspm install.
```
Anything else I can try?

**Comment by guybedford** [comment]
Can you share steps to reproduce here at all? I tried a simple `jspm install lodash && jspm bundle-sfx lodash` and that is working fine for me.

**Comment by lmartins** [comment]
Thank you Guy. Im not able to replicate the problem with another very similar projects im working on, so im closing this. If I found what causes the issue in this particular case ill post here.

**Comment by terasaka2k** [comment]
Steps to reproduce `EISDIR` when `defaultJSExtension` is `false`.

```bash
$ jspm init
$ jspm install 'reflect-metadata@0.1.2'
$ echo 'import "reflect-metadata";' > main.js
$ npm install jspm
$ node
```

```node
const jspm = require('jspm');
(new jspm.Builder({ defaultJSExtension: false })).bundle('main.js');
```

**Comment by guybedford** [comment]
@terasaka2k when downgrading from jspm 0.17 dev, you would need to run `jspm install` again.

**Comment by terasaka2k** [comment]
@guybedford 
The error went away. Thanks.

**Comment by vanthome** [comment]
Please reopen. At least with 0.16.18 I see this error for my project. When I downgrade to 0.16.15 it works. The problem is as there is no stack trace, I cannot tell which package it tries to install.

**Comment by guybedford** [comment]
@vanthome regression fixed in 0.16.19.

**Comment by Candy374** [comment]
same issue happen in 0.16.48
